.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,4 +14,8 @@
 
 ## Screenshots (if applicable)
 <!-- Include screenshots to demonstrate any UI changes. -->
-<!-- <img width="180" alt="image" src="image_url_here"> -->
\ No newline at end of file
+<!-- <img width="180" alt="image" src="image_url_here"> -->
+
+## Figma Link (if applicable)
+<!-- Add a link to the Figma design here if relevant to the PR. -->
+
```

Comment:
Rebase your branch with master, cuz you already made these changes in Template.

Reply:
fixed, thanks